robots/panther/ros_noetic/compose.rviz.yaml

```diff
@@ -0,0 +1,27 @@
+services:
+
+  rviz:
+    image: husarion/rviz:noetic
+    network_mode: host
+    ipc: host
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+      - ./panther.rviz:/root/.rviz/default.rviz
+    environment: 
+      - DISPLAY=${DISPLAY:?You need to define display env}
+      - ROS_MASTER_URI=${ROS_MASTER_URI:-http://10.15.20.2:11311}
+      - ROS_IP=${ROS_IP:-10.15.20.3}
+      - LIBGL_ALWAYS_SOFTWARE=1
+
+  remote-desktop:
+    image: donowak/remote-desktop:lan
```

Comment:
It would be nice if we could move it to the husarion organization

Reply:
The way of VNC server startup is changed now ;)